### DISCUSSION TITLE
Поднятие онлайна на ЦДБГ.

### DISCUSSION THREAD

**Discussion by HoRRiger** [octothorp-team/public-dobrograd#673] [Предложения]
Это будет как предложение, так и просьба. 

Все мы знаем, что уже больше месяца открыт ЦДБГ, который по «наследству» несёт проблему с онлайном. И это очень прискорбно, потому что сам сервер прикольный. На данный момент, создаётся мнение не только у меня, но и у других игроков, что сервер нужен только для Фарма.
 
 
Как я считаю, на сервер не заходят по-нескольким причинам: 1) Нет ничего интересного.
2) Когда видят, что на сервере +-10 человек, не заходят, потому что понимают, что им скучно будет провалить время на сервере.
3) Исходит из второго, нету никаких РП отношений.

Как я вижу, как можно исправить эту проблему - это узнать одну из главных причин, по которой игроки не хотят играть на ЦДБГ и в корне убрать ее. Или же, добавить, косвенно новое(Не могу толком предложить, что именно. Фантазия на уровне плинтуса) на сервер, потому что, видно, что (Не в обиду) разработчикам практически нет никакого дела до сервера(Например- та же недоработанная карта, а конкретно баги, где нет текстуры или же Хит-бокс больше чем объект в несколько раз.)

Теперь к просьбе. Поднимите этот вопрос, потому что сервер очень хороший и позволяет даже больше реализовать РП ситуаций чем на НДБГ. Иначе его придётся просто закрыть..(Мое мнение)

P.S. Данный текст не несёт за собой какой-либо агрессивный и оскорбительный посыл в сторону разработчиков и создателей проекта.

**Comment by SnakeLuck** [comment]
Да, получается так что люди заходят на ЦДБГ только чтобы пофармить руды и не более
